tests/main/lxd-refresh-cycle/task.yaml

```diff
@@ -0,0 +1,64 @@
+summary: Ensure that we can refresh/remove snaps in LXD
```

Comment:
This looks a bit similar to the existing tests/main/lxd - could we merge them?

Reply:
Well, yeah perhaps. I mainly wrote this with the specific goal of testing this bug. Perhaps we can just move it to regression and keep as is. I'd rather not loose this specific property because the test is optimized away somehow.

Reply:
Just add a comment to the stock lxd test saying what is being verified in the two or three lines below. That'd be nicer and more likely to remain useful and remain maintained overtime than 40 lines (!) cargo-culted. The property of being able to remove a snap is not special.

Reply:
Done